code/obj/medicaldiagnosis.dm

```diff
@@ -0,0 +1,91 @@
+/obj/item/medical/medicaldiagnosis
+	name = "UM EXCUSE ME WHY ARE YOU SEEING THIS PLEASE MAKE AN ISSUE AAAAA"
+	desc = "*scream"
+	icon_state = "stethoscope" // todo
+	inhand_image_icon = "stethoscope" // nuke this awful sprite later
+	icon = 'icons/obj/medicaldiagnosis.dmi' // NOO YOU CAN'T JUST FILL YOUR CODE WITH TODO MARKERS YOU NEED TO ACTUALLY DO SPRITING AND NOT JUST MAKE CODERSPRITES NOO
+// haha procrastination machine go brrr
+/obj/item/medical/medicaldiagnosis/stethoscope
+	name = "stethoscope"
+	desc = "a disc-shaped resonator attached to two earpieces for figuring out if someone has consumption or is simply suffering from the vapors."
+	attack(mob/living/carbon/human/M as mob, mob/user as mob)
```

Comment:
why is this typed to humans? attack() is on all mobs.

Reply:
added an ishuman check and pulled out the scope, this should be fixed.